<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Qcentris-396" failures="1" tests="2" name="ErrorValidationTest" time="7.244" errors="0" timestamp="2024-12-18T11:33:59 EET">
  <testcase classname="tests.ErrorValidationTest" name="testInvalidLoginLandingPage" time="0.801">
    <skipped/>
  </testcase> <!-- testInvalidLoginLandingPage -->
  <testcase classname="tests.ErrorValidationTest" name="testInvalidLoginLandingPage" time="0.513">
    <failure type="java.lang.AssertionError" message="expected [correct error message is Epic sadface: Password is required] but found [Epic sadface: Password is required]">
      <![CDATA[java.lang.AssertionError: expected [correct error message is Epic sadface: Password is required] but found [Epic sadface: Password is required]
at tests.ErrorValidationTest.testInvalidLoginLandingPage(ErrorValidationTest.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- testInvalidLoginLandingPage -->
</testsuite> <!-- ErrorValidationTest -->
